apps/passport-server/src/services/telegramService.ts

```diff
@@ -220,31 +266,55 @@ export class TelegramService {
           `Choose an event to manage.\n\n <i>✅ = this chat is gated by event.</i>`,
           {
             reply_markup: eventsMenu,
-            parse_mode: "HTML"
+            parse_mode: "HTML",
+            message_thread_id: messageThreadId
           }
         );
       } catch (error) {
-        await ctx.reply(`Error linking. Check server logs for details`);
+        await ctx.reply(`${error}`, { message_thread_id: messageThreadId });
         logger(`[TELEGRAM] ERROR`, error);
       }
     });
 
-    this.bot.command("help", async (ctx) => {
+    this.bot.command("setup", async (ctx) => {
+      const messageThreadId = ctx?.message?.message_thread_id;
+      try {
+        if (!isGroupWithTopics(ctx)) {
+          throw new Error("Pleae enable topics and try again");
+        }
+
+        if (ctx?.message?.is_topic_message)
+          throw new Error(`Cannot run setup from an existing topic`);
+
+        await ctx.editGeneralForumTopic(adminBotChannel);
+        await ctx.hideGeneralForumTopic();
+        const topic = await ctx.createForumTopic(`Announcements`, {
+          icon_custom_emoji_id: "5309984423003823246" // 📢
+        });
+        await ctx.api.closeForumTopic(ctx.chat.id, topic.message_thread_id);
+      } catch (error) {
+        await ctx.reply(`❌ ${error}`, {
+          reply_to_message_id: messageThreadId
+        });
+      }
+    });
+
+    this.bot.command("adminhelp", async (ctx) => {
```

Comment:
why not just "help"?

Reply:
I added help commands inline for users. Wanted something different for admins, so users don't have to see all the admin commands. wdyt?

<img width="363" alt="CleanShot 2023-10-01 at 18 54 02@2x" src="https://github.com/proofcarryingdata/zupass/assets/42984734/3729cc6b-9b76-4beb-9dbe-233c242d6d9a">

Reply:
can you make it so that non-admins don't see /adminhelp as an inline autocomplete?

Reply:
Doesn't look like that's possible for inline commands. Right now, I just didn't put adminHelp as an inline command. Admins who aren't us have basically no reason to use admin commands right now anyway. Just a couple items on setup

Reply:
I'll note this and we can revisit tho